Horse-racing Duals/index.test.js

```diff
@@ -0,0 +1,17 @@
+const index = require('./index');
```

Comment:
In unit test context one of many good practices is to have unified naming, because everywhere in the unit test file we are testing some "subject".
```suggestion
const testSubject = require('./index');
```